docs/source/getting_started/installing.rst

```diff
@@ -92,6 +99,22 @@ The ``vncauthproxy`` service is installed and can be started::
 
     $ sudo service vncauthproxy start
 
+systemd
+~~~~~~~
+
+For systems running ``systemd``, a basic systemd script has been provided by Adam at
+the Georgia Tech Information Security Center. It has been tested on Debian 8.
```

Comment:
He has not. If he doesn't respond to me asking on the issue, what would the best way to go about citing it be? Link to the issue? Or consider it trivial enough and not cite it?